spec/features/bucket_lists/user_creates_bucket_list_spec.rb

```diff
@@ -5,51 +5,43 @@
   I want to create a bucket list
   So I can later recall the destinations without weighing down my bag or mind.
 ) do
-# Acceptance Criteria
-# [x] I can create a new list from the index page
-# [x] I must provide a title
-# [x] If I do not provide a title, I cannot save it and recieve an error message
-# [x] By default, a list is private
-# [x] I may choose to make a list public
-# [x] I must click a button to create the list
-# [x] I recieve a success message when I successfully create a list
-# [x] I must be signed in to create the list
+#Acceptance Criteria
+#[x] I can create a new list from the index page
+#[x] I must provide a title
+#[x] If I do not provide a title, I cannot save it and recieve an error message
+#[x] By default, a list is private
+#[x] I may choose to make a list public
+#[x] I must click a button to create the list
+#[x] I recieve a success message when I successfully create a list
+#[x] I must be signed in to create the list
```

Comment:
Incorrect indentation detected (column 0 instead of 2).<br>Missing space after #.